cmd/config/validate.go

```diff
@@ -57,13 +59,14 @@ func NewValidateCmd() *cobra.Command {
 				return fmt.Errorf("failed to read config: %w", err)
 			}
 
-			return errors.Join(cfg.Validate()...)
+			return errors.Join(cfg.Validate(onlineFlag)...)
 		},
 		SilenceUsage: true,
 	}
 
 	cmd.PersistentFlags().AddFlagSet(config.GetPersistentFlagSet())
 	cmd.Flags().AddFlagSet(config.FileInputFlag())
+	cmd.Flags().BoolVar(&onlineFlag, "online", false, "Offline/online toggle. False by default")
```

Comment:
The idea with this toggle is to be able to run `k0s config validate` in ci/cd also. A place where connectivity to the storage may not be available.